tests/testcases/040_check-network-adv.yml

```diff
@@ -7,7 +7,7 @@
       args:
         executable: /bin/bash
       when:
-        - (calico_ipip_mode is defined and calico_ipip_mode != 'Never' or cloud_provider is defined)
```

Comment:
This seems to make the CI fail, guessing that the is defined should be kept or the whole `or cloud_provider` dropped

Reply:
Thank you! Let's try to remove it.